### ISSUE TITLE
Error when focus out of tab

### ISSUE THREAD

**Issue by chanthou** [avocode/avocode#1207]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.11.1`
Avocode UID: null
```
Error: Minified exception occurred; use the non-minified dev environment for the full error message and additional helpful warnings.
    at invariant (/Users/apple/Downloads/Avocode-darwin-x64/Avocode.app/Contents/Resources/app.asar/node_modules/react/node_modules/fbjs/lib/invariant.js:35:15)
    at Object.findDOMNode (/Users/apple/Downloads/Avocode-darwin-x64/Avocode.app/Contents/Resources/app.asar/node_modules/react/lib/findDOMNode.js:45:205)
    at t.a._focusElement (file:///Users/apple/Downloads/Avocode-darwin-x64/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:8:25008)
    at Immediate.<anonymous> (file:///Users/apple/Downloads/Avocode-darwin-x64/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:8:26731)
    at runCallback (timers.js:574:20)
    at tryOnImmediate (timers.js:554:5)
    at processImmediate [as _immediateCallback] (timers.js:533:5)
```

**Comment by vojtatranta** [comment]
Hi @chanthou,
thank you for this report. We know about this bug from Sentry and fix will be released in `2.12.0`.

**Comment by connor-bean** [comment]
Hey!

We have released the newest version of the Avocode app, version 2.12, which we expect to fix this problem. Please download it at https://avocode.com/download/. I am going to close this ticket but please comment on it if you’re still experiencing this problem in Version 2.12.

Thanks!
Connor, Avocode Support
